File: src/main/java/gestorAplicacion/servicios/Cliente.java
package gestorAplicacion.servicios;

import java.io.Serializable;

public class Cliente implements Serializable {
	
	//private static final long serialVersionUID = 1L;
	
	protected String tipoId;
	protected int id;
        protected int edad;
        protected Tarjeta tarjeta;
	protected Reserva reserva;

    public Cliente(String tipoId, int id, int edad) {
        this.tipoId = tipoId;
        this.id = id;
        this.edad = edad;
    }
        
        public int getEdad() {
            return this.edad;
        }
        
	public int getId() {
            return this.id;
	}
        
        public String getTipoId() {
            return this.tipoId;
	}
        
	public void corregir_Datos(String nueva_tipoId, int nueva_id) {
            this.tipoId = nueva_tipoId;
            this.id = nueva_id;
	}

    @Override
    public String toString() {
        return "Cliente{" + "tipoId=" + tipoId + ", id=" + id + ", edad=" + edad + ", reserva=" + reserva + '}';
    }

    
        
    
	
}

File: practica2/src/gestorAplicacion/servicios/cliente.py


class Cliente:

    def __init__(self, tipoId, id, edad):
        self._tipoId = tipoId
        self._id = id
        self._edad = edad
        self._tarjeta = None
        self._reserva = None
    
    def getReserva(self):
        return self._reserva
    
    def setTarjeta(self, tarjeta):
        self._tarjeta = tarjeta

    def setReserva(self, reserva):
        self._reserva = reserva

    def getEdad(self):
        return self._edad

    def getId(self):
        return self._id

    def getTipoId(self):
        return self._tipoId

    def corregirDatos(self, nueva_tipoId, nueva_id):
        self._tipoId = nueva_tipoId
        self._id = nueva_id

    def __str__(self):
        return f"Id: {self.getId()}, Edad {self.getEdad()}, Tarjeta: {self._tarjeta}, Reserva: {self._reserva}"

    


